alerts/activemq/metadata.yaml

```diff
@@ -5,16 +5,25 @@ alert_policy_templates:
   description: "When the broker's disk storage is near its limit(90%), then the producer
     flow control may cause producers to stop sending messages."
   version: 1
+  related_integrations:
```

Comment:
@EvanSimpson would it be easier to move this field on top of alert_policy_templates? i assume it would apply to all templates.

Reply:
We should do this on a per-template basis, to cover the possibility of multiple versions of a single integration requiring distinct alert templates.